src/tablist.js

```diff
@@ -18,6 +18,10 @@ SideTabList.prototype = {
     if (this.alwaysShrink) {
       this.maybeShrinkTabs();
     }
+
+    this.useRegExpFilter = (await browser.storage.local.get({
```

Comment:
I now understand your thoughts. But it looks needs to modify two PR to synchronize and focus on possible conflicts, should be reconstructed in the future?